configuration/forms.md

```diff
@@ -674,3 +674,44 @@ To use fauxton.
  1. Click add attachment
  1. Upload the multimedia file and ensure the name matches what has been defined in the form.
 
+## Contact forms
+
+These are forms that are used to create person or place types. Prior to version 3.7, CHT Core supported 4 contact types - 3 place types (clinic, health_center, district_hospital) and one person type (person). From version 3.7, we support an unlimited number of place and person types. Each contact-type should ideally have two forms; for creation and editing. 
+
+These forms are stored in the `forms/contact` subfolder of the project config directory. The naming convention used sould be `<contact_type_id-{create|edit}>.xlsx`. The `contact_type_id` prefix should match what's specified in the contact form settings page. 
+
+### Person-type forms
+
+These have the simplest structure since we are only creating one thing - the actual contact. Below, we'll look at the structure of the survey and settings sheets since these are critical to ensuring the contact is created correctly at the appropriate part of the hierarchy.
+
+#### Survey sheet
+
+![Person forms survey sheet](img/person-contact-form-survey.png)
+
+Section 1 and 3 are optional. We can pull in details of the logged in user as shown in section 1 and use that to log some metadata on the created documents as shown in section 3.
+
+Section 2 contains the core components of what will be saved in couchdb. The group name needs to match the contact type id specified in app_settings.json (if using the configurable hierarchy) or `person` if using the old-style hierarchy. To learn more about setting up configurable hieararchy, review the `Configuring the contact hierarchy` section in the [app-settings documentation](app-settings.md). `parent`, `type`, `contact_type` and `name` attributes are mandatory for things to work correctly.
+
+### Settings sheet
+
+![Person forms settings sheet](img/person-contact-form-settings.png)
+
+We'll need to replace `PLACE_NAME` with a name that corresponds to the created contact type. We'll also need to replace `PLACE_TYPE` with the contact-type id specified in app_settings.json (for configurable ones) or `person`.
+
+### Place-type forms
+
+The main difference between place type and person type forms is that we can optionally create one or more person-type documents once of which can be linked to the created place as a contact.
+
+We'll look at a simple structure of a place forms showing all the necessary components
+
+![Place forms survey sheet](img/place-contact-form-survey.png)
+
+Section 2 has specifies the contact that will be linked to the place being created. `parent`, `type` and `contact_type` and `name` are mandatory. This also applies to the place-type definition in section 4. `contact` on the other hand is not mandatory for the successful creation of a place. It usually more conventient to create a place and it's primary contact at the same time.
```

Comment:
The wording here is a little confusing. Section 2 specifies vs Section 2 has specifies.

Reply:
Added a note point its similarity to what has been described in person forms